<ui version="4.0" >
 <class>CSTextManageWidgetClass</class>
 <widget class="QWidget" name="CSTextManageWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CSTextManageWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLineEdit" name="fileName" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QToolButton" name="fileOpen" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLabel" name="count" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>17</height>
          </size>
         </property>
         <property name="text" >
          <string>Count</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GTableView" name="sourceView" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectColumns</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="" >
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QCheckBox" name="distinct" >
         <property name="text" >
          <string>Distinct</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="GTableView" name="targetView" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QProgressBar" name="progressBar" >
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>GTableView</class>
   <extends>QTableView</extends>
   <header>GTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
